rfcs/025-lineage-stage-0.md

```diff
@@ -29,11 +29,11 @@ Those implementing Amundsen should keep in mind that this feature is meant to pr
 
 ## UI/UX-level Explanation
 
-![Table Lineage Preview](assets/025/table-lineage-preview.png)
+![Table Lineage Preview](../assets/025/table-lineage-preview.png)
 
 We will add two additional tabs to the `Table Details` page, `Upstream` and `Downstream`. Each tab will contain a list of tables from which data is inherited or consumed. This allows users view a table's lineage in a very simple manner.
 
-![Column Lineage Preview](assets/025/column-lineage-preview.png)
+![Column Lineage Preview](../assets/025/column-lineage-preview.png)
```

Comment:
any reason we don't go with lineage tab with column and dashboard, then have upstream/downstream subtab underneath?

Reply:
I think there's a preference to avoid nested tabs when possible. At some point if we have too many tabs then we can reconsider nesting or other options.

Reply:
For me it feels weird to break it into two tabs. lineage is, after all, a big picture of data flow - so would make sense to have it in one place. It'd make it easier to analyze it for end user.

Reply:
yeah, +1 on @mgorsk1 , I felt the same to have single lineage tab with nested tab for upstream/downstream (not sure how complexity it changes for FE implementation). @danwom @allisonsuarez could you share more about the details of the preferences?

Reply:
+1 for not using the nested tabs where possible.

For Stage 0, it will be a list of items for each direction, which IMHO is okay to be placed in different tabs. But yes, for the next milestone, where we'll have a graph/chart of the complete lineage (and not the list of tables), that must be placed in one view under 1 tab to have a clear view of the complete lineage.

Reply:
I am not really advocating for nested tabs as this sounds more like implementation preference rather than UX question. Is nesting the only way this could be implemented in single tab ? I would even see something like single list (sorted by upstream and then downstream) with table names and icon/abbreviation if the table is upstream or downstream.